### ISSUE TITLE
Login screen should redirect to vault if user is already logged in

### ISSUE THREAD

**Issue by quinone** [quinone/password-manager#30]

**Comment by Rachie94** [comment]
Should it?  

should the user not be signed out if inactive or uses the back button? 
so should there be a time when they go back to login after being logged in?

**Comment by quinone** [comment]
Yeah I agree, but if your logged in and go to login screen are still logged in?

23 May 2024 13:52:51 Rachie94 ***@***.***>:
